compute/kubernetes/api-cli/creating-managing-kubernetes-lifecycle-cliv2.mdx

```diff
@@ -0,0 +1,322 @@
+  Do you want to install autocomplete? (Y/n):
+  ```
+5. Type `y` or `n` and hit enter.
+
+  ```
+  To enable autocomplete, scw needs to update your shell configuration.
+  What type of shell are you using (default: zsh):
+  ```
+
+6. Hit enter if the default shell type is already correct for you. Otherwise, enter your shell type and hit enter.
+
+  ```
+  To enable autocomplete we need to append to /Users/yourusername/.zshrc the following lines:
+  # Scaleway CLI autocomplete initialization.
+  eval "$(scw autocomplete script shell=zsh)"
+  Do you want to proceed with these changes? (Y/n):
+  ```
+
+7. Type `y` or `n` and hit enter. This should complete the configuration process:
+
+  ```
+  Initialization completed with success.
+  ```
+
+## Creating a K8s Kapsule cluster
+
+1. Type the following command in your terminal to create a [K8S Kapsule cluster](/compute/kubernetes/concepts/#cluster/):
+
+  `scw k8s cluster create name=name-of-your-cluster version=1.22.1 pools.0.name=name-of-your-pool pools.0.node-type=GP1_XS pools.0.size=5`
+
+  You will see an output similar to this: 
+
+  ```
+  The argument 'enable-dashboard' is deprecated, more info with: scw k8s cluster create --help
+  ID                12345678-1234-1234-4321-123456789123
+  Name              name-of-your-cluster
+  Status            creating
+  Version           1.22.1
+  Region            fr-par
+  OrganizationID    12345678-1234-1234-4321-123456789123
+  ProjectID         12345678-1234-1234-4321-123456789123
+  Cni               cilium
+  Description       -
+  ClusterURL        https://12345678-1234-1234-4321-123456789123.api.k8s.fr-par.scw.cloud:6443
+  DNSWildcard       *.12345678-1234-1234-4321-123456789123.nodes.k8s.fr-par.scw.cloud
+  CreatedAt         now
+  UpdatedAt         now
+  DashboardEnabled  false
+  Ingress           none
+  UpgradeAvailable  false
+
+  Autoscaler configuration:
+  ScaleDownDisabled              false
+  ScaleDownDelayAfterAdd         10m
+  Estimator                      binpacking
+  Expander                       random
+  IgnoreDaemonsetsUtilization    false
+  BalanceSimilarNodeGroups       false
+  ExpendablePodsPriorityCutoff   -10
+  ScaleDownUnneededTime          10m
+  ScaleDownUtilizationThreshold  0.5
+  MaxGracefulTerminationSec      600
+
+  Auto-upgrade settings:
+  Enabled                      false
+  MaintenanceWindow.StartHour  0
+  MaintenanceWindow.Day        any
+
+  Open ID Connect configuration:
+  IssuerURL       -
+  ClientID        -
+  UsernameClaim   -
+  UsernamePrefix  -
+  GroupsPrefix    -
+  ```
+
+  <Message type="note">
+
+  For more information about the creation of a K8s cluster, type `scw k8s cluster create --help` in your terminal. 
+
+  </Message>
+
+2. Write down the ID of your cluster as you will need it in the next steps. 
+
+
+## Installing a [KubeConfig](/compute/kubernetes/concepts/#kubeconfig/) configuration file 
```

Comment:
You can't put links in headers unfortunately, it breaks the side menu (on the right). Same for the next header.